### PR TITLE
[CI] Cleanup travis and CI steps

### PR DESCRIPTION
**Pull request by Djailla**
- Remove python 3.4 deprecated
- Cleanup Django / python compatibility matrix

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,44 +1,48 @@
 language: python
-# see https://github.com/travis-ci/travis-ci/issues/9069#issuecomment-425720905
-dist: xenial
```
Comment (antonagestam) [inline .travis.yml]: Does 3.7 really work without this?

File: .travis.yml [hunk 2]
```diff
@@ -1,44 +1,48 @@
...
+
```
[the remark this answers is not in the document]
Comment (Djailla) [reply .travis.yml]: Done

File: .travis.yml [hunk 3]
```diff
@@ -1,44 +1,48 @@
...
-    - env: DJANGO=2.1
-      python: '3.4'
-    - env: DJANGO=2.2
-      python: '3.4'
     - env: DJANGO=1.11
       python: '3.8'
     - env: DJANGO=2.1
       python: '3.8'
+    - env: DJANGO=2.2
+      python: '3.8'
```
[the remark this answers is not in the document]
Comment (Djailla) [reply .travis.yml]: Officially, Django 2.2 does not support Python 3.8 but agreed and fixed

File: .travis.yml (hunk 1 above)
Comment (Djailla) [reply .travis.yml]: It should, xenial is now the default configuration

**Comment by Djailla** [comment]
Hello @antonagestam, am I missing something here with the CI failing ?

**Comment by antonagestam** [comment]
@Djailla I'm scratching my head too, but haven't had time to look into it. Probably some issue with env vars in travis ...

**Comment by antonagestam** [comment]
Merged in #165

**Pull request closed without merging**